alsongDalsong/alsongDalsong/alsongDalsong/Sources/Components/UIKitComponents/ASButton.swift

```diff
@@ -2,120 +2,98 @@ import Combine
 import SwiftUI
 import UIKit
 
+/// 버튼의 동작을 관리
 final class ASButton: UIButton {
+    private var configurationData: ASButtonConfiguration?
     private var cancellables = Set<AnyCancellable>()
 
     init() {
         super.init(frame: .zero)
+        setupButton()
     }
 
     required init?(coder: NSCoder) {
         super.init(coder: coder)
+        setupButton()
     }
 
     /// 버튼의 UI 관련한 Configuration을 설정하는 메서드
     /// - Parameters:
     ///   - systemImageName: SF Symbol 이미지를 삽입을 원할 경우 "play.fill" 과 같이 systemName 입력.
     ///   - text: 버튼에 쓰일 텍스트
     ///   - textStyle: 버튼에 쓰일 텍스트 스타일
-    ///   - backgroundColor: UIColor 형태로 색깔 입력.  (ex) .asYellow)
+    ///   - backgroundColor: UIColor 형태로 색깔 입력.  (ex .asYellow)
```

Comment:
```suggestion
    ///   - backgroundColor: UIColor 형태로 색깔 입력.  (e.g. asYellow)
```

e.g. 를 쓰는게 더 맞는 표현 같습니다!! :)

Reply:
넵! 반영했습니다!